google/cloud/storage/internal/logging_client.cc

```diff
@@ -22,7 +22,7 @@ namespace storage {
 inline namespace STORAGE_CLIENT_NS {
 namespace internal {
 namespace {
-using namespace google::cloud::storage::internal::raw_client_wrapper_utils;
+using namespace raw_client_wrapper_utils;
```

Comment:
Not sure if I reviewed this before and didn't catch it, or if someone else did, but **IIUC**, this `using namespace` directive has some weird behavior, as described here:
https://stackoverflow.com/questions/10741428/does-using-namespace-cause-name-hiding

...resulting in the names in that namespace getting added to the "common namespace ancestor" (which looks to be `internal` in this case).  I'd suggest either using a namespace alias to avoid the long name (`namespace rcw_utils = ...`), or - probably better - adding a `using` directive for each name you're using (e.g. `using raw_client_wrapper_utils::CheckSignature`).

That being said, the `using namespace` directives were already present, so that change probably belongs in another commit -- I'll file an issue to track finding places where we do this and fixing them if you'd like.